evalml/pipelines/components/estimators/classifiers/decision_tree_classifier.py

```diff
@@ -0,0 +1,39 @@
+from sklearn.tree import DecisionTreeClassifier as SKDecisionTreeClassifier
+from skopt.space import Integer
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+
+
+class DecisionTreeClassifier(Estimator):
+    """Decision Tree Classifier."""
+    name = "Decision Tree Classifier."
+    hyperparameter_ranges = {
+        "criterion": ["gini", "entropy"],
+        "max_features": ["auto", "sqrt", "log2"],
+        "max_depth": Integer(4, 10)
```

Comment:
Is `max_depth` values just a placeholder for now until we do perf testing? It seems a little low off the top of my head.

Reply:
Ya I'm using the same value from our ExtraTrees components for now, don't know if there's anything better until we do some perf testing 😎